stl/inc/format

```diff
@@ -405,7 +431,7 @@ _NODISCARD constexpr const _CharT* _Parse_align(
 
 // Adapts a type modeling _Parse_spec_callbacks to model _Parse_arg_id_callbacks.
 // Used in _Parse_width so that _Parse_arg_id can be used to parse dynamic widths.
-template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+template <class _CharT, _Width_adapter_callbacks<_CharT> _Callbacks_type>

```

Comment:
Does the comment "`a type modeling _Parse_spec_callbacks`" need to be updated? Occurs below.